### PR TITLE
Pr@cag/enhance about page #25

### PR DESCRIPTION
**Pull request by CarlGraff**
Moved back button to upper left,  made more consistant with Landing page banner like format, and corrected some spelling errors.
Issue #25

### CODE REVIEW

**Comment by CarlGraff** [comment]
I forgot to add rounded as shown here:
 <div className="text-center **rounded** p-4 text-white bg-info" >
I can create anohter pull request or you can just edit it or perhaps there is a way form me to re-submit this PR.

**Comment by NickEngmann** [comment]
If you make the change on your end and push it to your branch it will automatically be added to the PR

File: p2plending/frontend/src/app/about/About.js [hunk 1]
```diff
@@ -2,34 +2,47 @@ import React, { Component } from "react";
 import overview from '../../assets/p2pOverviewDiagram.gif';
 import { Link } from "react-router-dom";
 
+const config = require("../../../config/index");
+const title = config.title;
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/about/About.js]: Good addition of the config.title!

File: p2plending/frontend/src/app/about/About.js [hunk 2]
```diff
@@ -2,34 +2,47 @@ import React, { Component } from "react";
...
+        <dl>
+          <dt>Overview:</dt>
+          <dd>- The P2P Library Exchange facilitates a way for Lenders to securely loan their books and other media to people who are interested in borowing these books and other media. The local library along with it's staff provides a means to safely and securley accomplih this goal as shown in the following diagram. </dd>
+          <p><img src={overview} className="img-fluid mx-auto" alt="Overview Diagram" /></p>
+          <dt>Current situation:</dt>
+          <dd>- There are individuals and companies that have collections of books and other media that in many case are just "gathering dust". Meanwhile these books and 
+            media could enrich the lives and enjoyment of others.</dd>
+          <dt>An opportunity to help:</dt>
+          <dd>- We believe many individuals and companies will glady share their collections given an easy and safe way to so. Individuals wishing to borrow these shared collections in turn desire an easy way to locate and borrow this shared media.</dd>
+          <dt>How the P2P Libary Lending system "Lendo" helps:</dt>
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/about/About.js]: I'm also going to go ahead and replace the Lendo here with the { title } variable

File: p2plending/frontend/src/app/about/About.js [hunk 3]
```diff
@@ -2,34 +2,47 @@ import React, { Component } from "react";
...
-      </dl>
+        <span role="img" aria-label="emoji" style={{ fontSize: 50 }}>
+          📖
+        </span>
+          <h2 className="m-0">{ title }</h2>
+        <div className="mx-auto" style={{ maxWidth: "500px" }}>
+          <span>
+            {title} is a Peer to Peer Book Exchange! Share and explore the books within your local community.{" "}
+            <span role="img" aria-label="Globex emoji">
+              🌎
```
Comment (NickEngmann) [inline p2plending/frontend/src/app/about/About.js]: 💙 the emoji addition

**Pull request merged**